src/dropdown/dropdown.component.ts

```diff
@@ -25,7 +25,6 @@ import {
 
 import { AbstractDropdownView } from "./abstract-dropdown-view.class";
 import { I18n } from "./../i18n/i18n.module";
-import { ListItem } from "./list-item.interface";
```

Comment:
This may be needed to solve a compiler issue ... I'd leave it just in case 😬 maybe add a comment though